lib/errors.js

```diff
@@ -264,7 +242,8 @@ exports.ConnectionRefusedError = ConnectionRefusedError;
 /**
  * Thrown when a connection to a database is refused due to insufficient privileges
  * @extends ConnectionError
- * @constructor
+ * @memberof Errors
+ * @memberof Sequelize
```

Comment:
Imo it should be enough to note that they are availbable in the Errors namespace description. Otherwise we would clutter up the properties.